conf/modules/optical_flow_landing.xml

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="optical_flow_landing" dir="ctrl">
+  <doc>
+    <description>
+        Optical flow landing.
+        
+        This module implements optical flow landings in which the divergence is kept constant. When using a fixed gain for control, the covariance between thrust and divergence is tracked, so that the drone knows when it has arrived close to the landing surface. Then, a final landing procedure is triggered. It can also be set to adaptive gain control, where the goal is to continuously gauge the distance to the landing surface. In this mode, the drone will oscillate all the way down to the surface. 
```

Comment:
Would probably be good to mention here that this is for rotorcrafts/multicopters and not fixedwings?